app/assets/stylesheets/components/_blue-box-campaign.scss

```diff
@@ -0,0 +1,45 @@
+$light-grey: #bfc1c3;
+$blue: #005ea5;
+$white: #ffffff;
+
+.app-c-native-blue-box-campaign__border {
+  display: block;
+  border-top: 1px solid $light-grey;
+  padding: 20px 0 0;
+  text-align: center;
+}
+
+.app-c-native-blue-box-campaign__link {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  background: $blue;
+  width: 100%;
+  min-height: 280px;
+  color: $white;
+  text-decoration: none;
+  text-align: left;
+  box-sizing: border-box;
+  padding: 15px;
+  font-size: 27px;
```

Comment:
I would suggest trying to use GOV.UK Elements font mixins, e.g: @include bold-24 in this case. I think this includes the line-height.